dask_sql/physical/rel/custom/create_model.py

```diff
@@ -125,10 +125,35 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
             raise ValueError("Parameters must include a 'model_class' parameter.")
 
         target_column = kwargs.pop("target_column", "")
-        wrap_predict = kwargs.pop("wrap_predict", False)
-        wrap_fit = kwargs.pop("wrap_fit", False)
+        wrap_predict = kwargs.pop("wrap_predict", None)
+        wrap_fit = kwargs.pop("wrap_fit", None)
         fit_kwargs = kwargs.pop("fit_kwargs", {})
 
+        try:
+            ModelClass = import_class(model_class)
+        except ImportError:
+            raise ValueError(
+                f"Can not import model {model_class}. Make sure you spelled it correctly and have installed all packages."
+            )
```

Comment:
I realize this is a copy of the existing error we had here, but standardizing on raising the right error where possible.
```suggestion
            raise ImportError(
                f"Failed to import model {model_class}. Make sure it is spelled correctly and the relevant packages are installed."
            )
```